### ISSUE TITLE
Remove signing logic from and rename "getOrder" call (order-utils)

### ISSUE THREAD

**Issue by dmosites** [airswap/airswap-protocols#280]
Would like to simplify the `getOrder` function and consider renaming it to `createOrder` or similar.

**Comment by alicevhenshaw** [comment]
I agree. Creating the order, and getting the signature should be 2 different things. One you do using orders.js, the other you do using signatures.js. It would lead to a nice separation of concerns and decrease the code complexity

**Comment by ejwessel** [comment]
implemented in #284
